psydac/api/tests/test_2d_multipatch_poisson.py

```diff
@@ -102,9 +102,9 @@ def test_poisson_2d_2_patch_dirichlet_1():
     A = Square('A',bounds1=(0, 0.5), bounds2=(0, 1))
     B = Square('B',bounds1=(0.5, 1.), bounds2=(0, 1))
 
-    domain = A.join(B, name = 'domain',
-                bnd_minus = A.get_boundary(axis=0, ext=1),
-                bnd_plus  = B.get_boundary(axis=0, ext=-1))
+    connectivity = [((0,0,1),(1,0,-1))]
+    domains = [A,B]
+    domain = Domain.join(domains, connectivity, 'domain')
```

Comment:
As above.

Reply:
done in [d66a68e](https://github.com/pyccel/psydac/pull/309/commits/d66a68ed7d030e41dc86823c805ed685b78d906e)